third_party/transducer/CMakeLists.txt

```diff
@@ -1,20 +1,7 @@
-CMAKE_MINIMUM_REQUIRED(VERSION 3.5)
-
-PROJECT(rnnt_release)
-
-SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -O2")
-
-IF(APPLE)
-    ADD_DEFINITIONS(-DAPPLE)
-ENDIF()
-
-INCLUDE_DIRECTORIES(submodule/include)
-
-SET(CMAKE_POSITION_INDEPENDENT_CODE ON)
```

Comment:
PIC argument has to be consistent across all the libraries. The current configuration works without this explicit line, but I moved this to the top level `CMakeLitst.txt`